### DISCUSSION TITLE
Kicking Members from Channel using Python

### DISCUSSION THREAD

**Discussion by sanjay-deoram** [python-telegram-bot/python-telegram-bot#3866] [Q&A]
I want to be able to kick members from the channel using my bot how do I go about doing that . Also im unable to use 

from telegram import Bot

bot  = Bot(token)

Gives me a empty Bot()

I doubled check my api key and everything

**Comment by Bibo-Joshi** [comment]
What do you mean by an "empty" (or "full"?) Bot?
Please be sure to check out the https://github.com/python-telegram-bot/python-telegram-bot/wiki/Introduction-to-the-API as well as the docs of https://docs.python-telegram-bot.org/telegram.bot.html#telegram.Bot.ban_chat_member
